Formula/libxml2.rb

```diff
@@ -41,7 +41,7 @@ class Libxml2 < Formula
 
   keg_only :provided_by_osx
 
-  depends_on :python => :optional
+  depends_on :python if MacOS.version <= :snow_leopard
```

Comment:
Mildly weird that `ssh` still ships with its own super-recent LibreSSL when the system-wide one is `2.2.7`. Maybe that'll change as we edge towards release slowly.

Although not sure how much to read into the version number at this point given on macOS Sierra `openssl` self-declares as `0.9.8zh` but Apple has acknowledged in previous security update notes it's actually kind of a mutant with security fixes plucked off of newer releases.